content/en/docs/Tasks/install-operator-with-olm.md

```diff
@@ -51,4 +51,4 @@ NAME                         READY   UP-TO-DATE   AVAILABLE   AGE
 <name-of-your-operator>      1/1     1            1           9m48s
 ```
 
-If the ClusterServiceVersion fails to show up or does not reach the `Succeeded` phase, please check the [troubleshooting documentation](openshift/coming-soon.md)) to debug your installation. 
\ No newline at end of file
+If the ClusterServiceVersion fails to show up or does not reach the `Succeeded` phase, please check the [troubleshooting documentation](/docs/contribution-guidelines/coming-soon/) to debug your installation. 
```

Comment:
We do have a troubleshooting doc for this: https://github.com/operator-framework/olm-docs/blob/master/content/en/docs/Tasks/Troubleshooting/clusterserviceversion.md